src/banking_stage.rs

```diff
@@ -110,7 +119,22 @@ impl BankingStage {
 
             debug!("process_transactions");
             let results = bank.process_transactions(transactions);
-            let transactions = results.into_iter().filter_map(|x| x.ok()).collect();
+            let transactions = results
+                .into_iter()
+                .filter_map(|x| match x {
+                    Err(BankError::AccountNotFound(tx)) => {
```

Comment:
I would benefit from an explanation of what's happening here, or what would happen without this code.

Reply:
The bank is returning error when vote is for an old transaction. The error now includes the transaction (vote) information. The code change checks if the transaction was a vote. It adds the nodes to the list of nodes that are voting. The leader will use this new node for broadcast henceforth.